src/cfml/system/services/ServerService.cfc

```diff
@@ -1589,6 +1587,7 @@ component accessors="true" singleton {
 			menuItem[ 'shell' ] = menuItem.shell ?: fileSystemUtil.getNativeShell();
 			// Some special love for box commands
 			if( menuItem.command.lCase().reFindNoCase( '^box(\.exe)? ' )  ) {
+				menuItem.command = fixBinaryPath( trim(menuItem.command), systemSettings.getSystemSetting( 'java.class.path' ));
```

Comment:
FWIW, this may be ok, but you are no longer accounting for the possibility `java.class.path` doesn't exist or is empty.